commcare_connect/reports/views.py

```diff
@@ -359,15 +359,32 @@ def __init__(self, *args, **kwargs):
             ),
             Row(
                 Column("delivery_type", css_class="col-md-4"),
-                Column("year", css_class="col-md-4"),
-                Column("month", css_class="col-md-4"),
+                Column("from_date", css_class="col-md-4"),
+                Column("to_date", css_class="col-md-4"),
             ),
-            Row(Column("by_delivery_type", css_class="col-md-4")),
+            Row(Column("group_by_delivery_type", css_class="col-md-4")),
         )
 
+        if not self.data:
```

Comment:
what is this doing and why is it now necessary?

Reply:
This is used to configure the filter's defaults. It detects that the filter was not set in a request and gives it a default from and to date.